### ISSUE TITLE
Focus styling for CardLinks

### ISSUE THREAD

**Issue by bryophyta** [guardian/dotcom-rendering#5177]
In the onwards carousel-type component on Fronts, there is no visible focus styling for the article cards, which makes keyboard navigation difficult. At the same time, using the keyboard one is able to highlight and click through to a `#comments` link for an article, even if comments are not enabled and there's no visible link. (Screenshot below is of the comments link being focused -- small blue square in the bottom right corner of the middle card.)

![Image](https://user-images.githubusercontent.com/37048459/173330816-e4b78d56-b115-4c58-8f74-5d48bfeb5a1b.png)

- [x] Visible focus highlights should be added to the cards (existing hover styling is not very pronounced, so might be better to use something else, e.g. the focus ring from Source?).
- [x] #5634
